src/views/Player.vue

```diff
@@ -240,6 +240,19 @@ export default defineComponent({
       clearInterval(_intervalRefreshHandle);
     }
 
+    onMounted((): void => {
+      // Since # is a reserved character for the "fragment/hash" part of the URL
+      // we can try to redirect to the correct URL if the hash is a battle tag,
+      // (starts with at least 4 digits). It may also be followed by a tab name,
+      // for example Someone#9999/matches would redirect to Someone%239999/matches
+      if (window.location.hash.match(/^#\d{4}/)) {
+        const url = new URL(window.location.href);
+        url.pathname = `${url.pathname}${url.hash}`;
```

Comment:
And this will automatically be escaped when we copy over the # ?

Reply:
Yep! Browser magic, automatically re-encodes.